src/app/foundations/ai/ai-demo.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="demo-content">
+    <h1>Healthcare.AI™ Branding</h1>
+    <h6>Last updated March 3, 2023</h6>
+
+    <hc-tile>
+        <h5 id="overview">Overview</h5>
+
+        <p>
+            <a target="_blank" href="https://healthcare.ai/">Heathcare.AI™</a> is HCAT's AI framework that expands the use and use cases for effective AI for our customers.
+
+        <p>In order to call out and tie together the uses of AI in our products, we recommend using standard visual branding.
+            When someone sees this branding we want them to think <span class="hc-font-em">"Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI."</span>
```

Comment:
Replace

`"Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI."`

with

`&quot;Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI.&quot;`